be/src/exec/schema_scan_node.cpp

```diff
@@ -158,8 +159,8 @@ Status SchemaScanNode::prepare(RuntimeState* state) {
         // TODO(zhaochun): Is this slow?
         int j = 0;
         for (; j < _src_tuple_desc->slots().size(); ++j) {
-            if (boost::iequals(_dest_tuple_desc->slots()[i]->col_name(),
-                               _src_tuple_desc->slots()[j]->col_name())) {
+            if (iequal(_dest_tuple_desc->slots()[i]->col_name(),
```

Comment:
maybe we can try to remove boost include

Reply:
done